pest_meta/src/parser.rs

```diff
@@ -57,19 +55,19 @@ pub enum GrammarRule {
     single_quote
 }
 
-pub struct GrammarParser;
+pub struct PestParser;
 
-impl Parser<GrammarRule> for GrammarParser {
+impl Parser<PestRule> for PestParser {
     fn parse<I: Input>(
-        rule: GrammarRule,
+        rule: PestRule,
         input: Rc<I>
-    ) -> Result<Pairs<GrammarRule, I>, Error<GrammarRule, I>> {
+    ) -> Result<Pairs<PestRule, I>, Error<PestRule, I>> {
         fn grammar_rules<I: Input>(
             pos: Position<I>,
-            state: &mut ParserState<GrammarRule, I>
+            state: &mut ParserState<PestRule, I>
         ) -> Result<Position<I>, Position<I>> {
             pos.sequence(|pos| {
-                soi(pos, state).and_then(|pos| {
```

Comment:
I was actually going to ask about this one, because I wasn't sure if it was there for historical reasons or if it was written in expectation of an upcoming feature. Is it OK that I took it out?

Reply:
Yup!